auth_server_example/sample.rb

```diff
@@ -0,0 +1,16 @@
+require 'sinatra'
+require 'json'
+require 'cgi'
+require_relative '../lib/pusher-platform'
+
+# Get these from the Dashbaord
+instance = PuPusherPlatformsher::Instance.new(
```

Comment:
Typo? Would be nice if those examples were run in CI.

Reply:
Most certainly a typo - thanks.

We'll get to having CI someday, but not worth it right now IMO